### DISCUSSION TITLE
All transactions are stuck.

### DISCUSSION THREAD

**Discussion by gymstreet** [filecoin-project/community#681] [Developers]
Hey guys, i need help, we recently integrated FIL to our product, and our transactions are not going through.

https://filfox.info/en/address/f410fahjdprwwq2heqpyf6o3zcjnlmj2vbshgq23dkqy?t=0

More than 3 hours it's saying PENDING. what am i doing wrong? on testnet it was working fine. on mainnet it is just stuck, and i don't know what to do. 

I have tried to adjust gas fee, also adjusted Gas Premium, didn't work.
Used metamask transaction - > didn't work
Used Glif's FilForwarder -> didn't work

**Comment by wolfduan** [comment]
github

**Reply by gymstreet** [reply]
what this suppose to mean ?
